### DISCUSSION TITLE
求问：程序里怎么获取当前的Apollo配置的apollo.meta

### DISCUSSION THREAD

**Discussion by daanhan** [apolloconfig/apollo#4291] [General]
@Value("${apollo.meta}") 这个形式没法获取到。
我希望获取到/opt/settings/server.properties里的 apollo.meta，也就是Apollo的服务地址，准备用来初始化ApolloOpenApiClient。谢谢！

**Comment by nobodyiam** [comment]
see com.ctrip.framework.apollo.util.ConfigUtil#getMetaServerDomainName.
but what ApolloOpenApiClient needs is the portal address instead of the meta server address, I'm not sure whether it's useful for you.
